tests/StacktraceTest.php

```diff
@@ -138,54 +138,47 @@ public function testAddFrameMarksAsInApp(): void
         $this->assertTrue($frames[1]->isInApp());
     }
 
-    public function testAddFrameReadsCodeFromShortFile(): void
+    /**
+     * @dataProvider contextDataProvider
+     */
+    public function testAddFrameWithDifferentContexts($fixture, $lineNumber, $contextLines, $preContextCount, $postContextCount)
```

Comment:
Please rename this test method to `testAddFrameRespectsContextLinesOption`

Reply:
Done